usefulUtilities/statusIndicator/README.md

```diff
@@ -60,19 +56,30 @@ Here's what your `config.json` file should look like:
     "mySQLUsername": "<MySQL DB username>",
     "mySQLPassword": "<MySQL DB password>",
     "databaseName": "<the name of the database associated with the Status Indicator app>",
-    "wwwRoot": "<a URL to the remote host serving the files in `/webApp/www`. include the trailing slash. used for testing so that the `canary` API endpoint knows where to check. example: https://example.com/statusIndicator/>"
+    "wwwRoot": "<a URL to the remote host serving the files in `/webApp/www/`. include the trailing slash. used for testing so that the `canary` API endpoint knows where to check. example: https://example.com/statusIndicator/>"
```

Comment:
```suggestion
    "wwwRoot": "<a URL to the remote host serving the files in `/webApp/www/`. Include the trailing slash. Used for testing so that the `canary` API endpoint knows where to check. Example: https://example.com/statusIndicator/>"
```

Reply:
stylistically, i'm keeping all of these sentences lower-case. i don't have a super good reason for it, but i'm consistent :)